GramAddict/core/bot_flow.py

```diff
@@ -48,14 +48,17 @@
 TESTED_IG_VERSION = "202.0.0.37.123"
 
 
-def start_bot():
+def start_bot(**kwargs):
     # Pre-Load Config
-    configs = Config(first_run=True)
+    if not kwargs:
+        configs = Config(first_run=True)
+    else:
+        configs = Config(first_run=True, module=True, **kwargs)
 
     # Logging initialization
     configure_logger(configs.debug, configs.username)
     logger = logging.getLogger(__name__)
-    if "--config" not in configs.args:
+    if "--config" not in configs.args and "config" not in configs.args:
```

Comment:
if "config" not in configs.args:
works for --config and config

Reply:
I fix that